packages/website/pages/docs.js

```diff
@@ -15,6 +16,12 @@ import DocsNavContent from '../components/navigation/docs-nav-content';
 import NavigationWrapper from '../components/navigation-wrapper';
 import PageTitle from '../components/page-title';
 
+const NoDocsMessage = () => (
+  <SectionMessage appearance="warning">
+    There do not currently exist any docs pages.
```

Comment:
Can we change this to something like "we couldn't find any docs pages in your provided docs directory"

(feel free to throw in some emojis to make it personable as well)